pkg/controller/installation/core_controller.go

```diff
@@ -1354,6 +1359,37 @@ func (r *ReconcileInstallation) validateTyphaCAConfigMap() (*corev1.ConfigMap, e
 	return cm, nil
 }
 
+// setDefaultOnFelixConfiguration will take the passed in fc and add any defaulting needed
+// based on the install config. If create is true then the FelixConfig default will be created,
+// otherwise a patch will be performed.
+func (r *ReconcileInstallation) setDefaultsOnFelixConfiguration(ctx context.Context, install *operator.Installation, fc *crdv1.FelixConfiguration, create bool, log logr.Logger) error {
+	// If we're using the AWS CNI plugin we need to ensure the route tables that calico-node
+	// uses do not conflict with the ones the AWS CNI plugin uses so default them
+	// in the FelixConfiguration if they are not already set.
+	if install.Spec.CNI.Type == operator.PluginAmazonVPC {
+		if fc.Spec.RouteTableRange == nil {
+			patchFrom := client.MergeFrom(fc.DeepCopy())
+			fc.Spec.RouteTableRange = &crdv1.RouteTableRange{
+				Min: 31,
```

Comment:
Is this range correct - curious to understand where these numbers are derived from.

Depending on which version of the VPC CNI plugin I'm looking at, I see table indexes being assigned based on:
- The ENI device number + 1   (and I think we can have up to 4x15 devices for p4d.24xlarge)
- The VLAN table ID + 100

Reply:
^^^ it's also possible that my code searching in the plugin was flawed... so I could be totally out there!

Reply:
Also, worth including any details in the comment here about this range.